packages/web/src/config.ts

```diff
@@ -5,3 +5,10 @@ if (typeof __REDWOOD__API_PROXY_PATH !== 'undefined') {
   // @ts-expect-error-next-line
   global.__REDWOOD__API_PROXY_PATH = __REDWOOD__API_PROXY_PATH
 }
+/* eslint-disable no-undef */
+// The value is read from `redwood.toml`
+// @ts-expect-error This is replaced at build time by Webpack Define Plugin.
+if (typeof __REDWOOD__APP_TITLE !== 'undefined') {
```

Comment:
Don't think we need this check here, because it'll always be defined

Reply:
Yes, But I got a TS error if the check is not there. I don't really understand why, I will try again just to be sure.

Reply:
Seem to work without now